### DISCUSSION TITLE
Character-by-character editing

### DISCUSSION THREAD

**Discussion by Bad-Assumptions** [VasilKalchev/LiquidMenu#102] [Ideas]
Thanks for this library. Looks logically organized and lightweight. 

I'd like to do the same thing mentioned in topic #55. Any changes or new thoughts since then?

**Comment by VasilKalchev** [comment]
Hello @Bad-Assumptions.

There haven't been development in that direction. I'll note that there is some demand for a feature like that, but I'm not currently able to develop new features.

Feel free to open a pull request if you implement the feature in your fork.

**Comment by Bad-Assumptions** [comment]
Thanks for the reply! I did manage to find a solution, though I don't recall the details.
